templates/support/index.html

```diff
@@ -12,7 +12,7 @@
   <div class="row u-equal-height">
     <div class="col-7">
       <h1>Ubuntu Advantage commercial support</h1>
-      <p>Ubuntu Advantage is the commercial support package from Canonical. It includes Landscape, the Ubuntu systems management tool, and the Canonical Livepatch Service, which enables you to apply kernel fixes without restarting your Ubuntu {{lts_release_full}} systems.</p>
+      <p>Canonical offers a range of Ubuntu Advantage packages to support and add value to your Ubuntu deployment. Ubuntu Advantage typically include Landscape, the Ubuntu systems management tool for security audit and compliance, and the Canonical Livepatch Service, which enables you to apply kernel fixes without restarting your {{lts_release_full}} systems</p>
```

Comment:
Not your fault but this should have a full stop at the end.

Reply:
Copt doc says `restarting your Ubuntu 16.04 LTS systems`

Reply:
My bad. Should be fixed now